mypy.ini

```diff
@@ -104,13 +105,13 @@ disable_error_code = arg-type,no-untyped-call,no-untyped-def
 disable_error_code = no-untyped-call,no-untyped-def
 
 [mypy-plasmapy.analysis.time_series.excess_statistics]
-disable_error_code = no-untyped-call,no-untyped-def
+disable_error_code = no-untyped-call,no-untyped-def,var-annotated
```

Comment:
I suspect new errors are being reported because untyped functions & methods were not previously being type checked.